htdocs/emailcollector/class/emailcollector.class.php

```diff
@@ -679,7 +679,7 @@ public function getConnectStringIMAP($ssl = 1, $norsh = 0)
         $flags = '/service=imap'; // IMAP
         if ($ssl) $flags .= '/ssl'; // '/tls'
         $flags .= '/novalidate-cert';
-        //$flags.='/readonly';
+        $flags.='/readonly';
```

Comment:
If we set this flag, i'm afraid that the feature that move the email into the "Mailbox target directory" once processed will won't work anymore, won't it ?